autoscaler/controllers/gateway/config/causely.go

```diff
@@ -0,0 +1,44 @@
+package config
```

Comment:
#1195  moved the destination configurators into `common` from `autoscaler`.

To fix the compilation issues, this file should be moved there as well